src/lib/dates.ts

```diff
@@ -44,3 +47,22 @@ export const parseDateTimePickerValue = (
   timeZone: string
 ): DateTime =>
   DateTime.fromISO(value).setZone(timeZone, { keepLocalTime: true })
+
+/**
+ * Formats a date time string to an object with date and time properties.
+ */
+export const formatDateTime = (
```

Comment:
Right now, [each component defines it's own date and time format locally](https://github.com/seamapi/react/blob/1221de8024782ab974d40a2a663bf870137c36c2/src/lib/seam/components/ClimateSettingScheduleTable/ClimateSettingScheduleRowDetails.tsx#L51-L55). If we could standardize all components to share the same format, then I agree we should have three functions in this file: `formatDate`, `formatTime`, an `formatDateTime`.

Since this is only used in one place, let's move it to the file that uses it (src/lib/ui/noise-sensor/NoiseSensorEventItem.tsx), and simplify the usage

```ts
const formatDate = (dateTime: string) => DateTime.fromISO(dateTime).toFormat(ABBREVIATED_DATE_FORMAT)

const formatTime = (dateTime: string) => DateTime.fromISO(dateTime).toFormat(TIME_FORMAT)
```

Reply:
Gotcha, okay!